app/views/content/landing/how-to-fund-your-teacher-training/_content.html.erb

```diff
@@ -100,7 +100,7 @@
 
       <ul>
         <li>£331.95 a week to support childcare costs for 2 or more children</li>
-        <li>£3,438 a year if an adult depends on you</li>
+        <li>$finance_adultdependantsgrant_value$ a year if an adult depends on you</li>
```

Comment:
This hasn't rendered properly on the front-end @MylesJarvis (because it is in a `li`?)

<img width="425" alt="image" src="https://github.com/DFE-Digital/get-into-teaching-app/assets/87642394/cdb77eff-32d0-4902-b9af-0cbf3cf7af82">